tests/jq_fuzz_execute.cpp

```diff
@@ -16,6 +16,10 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t *data, size_t size) {
   jq_state *jq = NULL;
   jq = jq_init();
   if (jq != NULL) {
+    char *origin = strdup("/tmp/");
+    jq_set_attr(jq, jv_string("JQ_ORIGIN"), jv_string(origin));
+    free(origin);
```

Comment:
Could this be `jq_set_attr(jq, jv_string("JQ_ORIGIN"), jv_string("/tmp/));`? skip strdup/free

Reply:
Yes, thanks! Done